src/rdkafka.c

```diff
@@ -551,7 +553,10 @@ void rd_kafka_get_err_descs (const struct rd_kafka_err_desc **errdescs,
 
 const char *rd_kafka_err2str (rd_kafka_resp_err_t err) {
 	static RD_TLS char ret[32];
-	int idx = err - RD_KAFKA_RESP_ERR__BEGIN;
```

Comment:
i don't understand the rational for this change (same comment below). won't it just force extra type casting with no actual effect?

Reply:
:+1:
Other part of the code erroneously stored a int16_t in an int, resulting in invalid error codes, but that has been fixed so this is no longer neede. Reverting